CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 TiUP Changelog
 
+## [1.3.1] 2020.12.31
+
+### Fixes
+
+- Workaround the issue that store IDs in PDs are not monotonically assigned ([#1011](https://github.com/pingcap/tiup/pull/1011), [@AstroProfundis](https://github.com/AstroProfundis))
```

Comment:
It's better to note when would the PD report non-monotonical store IDs, as it's not a very common circumstance to occur.